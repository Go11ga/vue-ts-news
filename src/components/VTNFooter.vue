<template>
    <footer class="footer">
        
        <div class="footer__credentials">
            by Igor Khromov Â©
        </div>
                
    </footer>
</template>

<script lang="ts">
import { defineComponent } from 'vue'

export default defineComponent({
    name: 'VTNFooter'
})
</script>
