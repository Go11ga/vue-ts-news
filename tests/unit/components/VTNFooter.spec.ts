import { shallowMount } from '@vue/test-utils'
import VTNFooter from '@/components/VTNFooter.vue'

describe('VTNFooter', () => {
    let wrapper = shallowMount(VTNFooter)

    it('dummy name test', () => {
        expect(wrapper.vm.$options.name).toMatch('VTNFooter')
    })

    it('snapshot', () => {
        expect(wrapper.element).toMatchSnapshot()
    })
})
