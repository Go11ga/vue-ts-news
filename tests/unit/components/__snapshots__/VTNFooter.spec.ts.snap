// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VTNFooter snapshot 1`] = `
<footer
  class="footer"
>
  <div
    class="footer__credentials"
  >
     by Igor Khromov Â© 
  </div>
</footer>
`;
